scripts/copy-styles.js

```diff
@@ -9,7 +9,7 @@ function replace(file, search, replacement) {
     }
 }
 
-function getInstalledComponents() {
+function getInstalledWebcomponentsDirnames() {
     let installedComponents = [];
     fs.readdirSync('../node_modules').forEach((file) => {
```

Comment:
Triviaal, maar file misschien hernoemen naar dir, in dit geval zijn het folders en geen bestanden?

Reply:
Akkoord, aangepast.